SEED-platform/seed
# Patch/mapping controller

#### Any background context you want to provide?
I broke some things...
#### What's this PR do?
Fixes the following:
1. importing tax-lots would result in the user getting continually redirected to the data mapping page. This should no longer happen
2. importing a BuildingSync file with an invalid schemaLocation, while celery was on and 'eager evaluation' was off, would result in an exception to be thrown in the task itself, and the user getting stuck on the import file dialog (this bug was discovered while fixing the one above so I just included the fix here)
#### How should this be manually tested?
1. import example spreadsheet and set all column types to TaxLot. You should be able to import the data without an issue
2. import a buildingsync file without a schema location or an invalid one. you should immediately get an error message stating such

## NOTE
This issue (no mapped properties) will become irrelevant when #2189 is merged, as it defers data quality checks to the data quality checker and the user. So before that PR is merged we should remove these checks for `hasMappedModels`.
